### PR TITLE
Introduce dev version branch and label

### PR DESCRIPTION
**Pull request by abelhegedus**
Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [x] Bumped the build number (if the version is unchanged)
* [x] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by abelhegedus** [comment]
@conda-forge-admin, please rerender

**Comment by abelhegedus** [comment]
@conda-forge-admin, please rerender

**Comment by abelhegedus** [comment]
Checks are failing due to missing API client dev version on conda-forge (I think).

**Review by FuzesiMate — APPROVED** [review]

**Comment by FuzesiMate** [comment]
@conda-forge-admin, please rebuild

**Comment by FuzesiMate** [comment]
@conda-forge-admin, please restart ci

**Comment by FuzesiMate** [comment]
@conda-forge-admin, please restart ci

File: conda-forge.yml [hunk 1]
```diff
@@ -1 +1,4 @@
+channels:
+  targets:
+  - [conda-forge, dev]
```
Comment (FuzesiMate) [inline conda-forge.yml]: Indentation might be wrong:
```suggestion
    - [conda-forge, dev]
```

**Review by FuzesiMate — CHANGES_REQUESTED** [review]

File: conda-forge.yml (hunk 1 above)
Comment (abelhegedus) [reply conda-forge.yml]: I guess we should setup dev label on the api client and migrate a working solution here when ready.

**Pull request closed without merging**